ansible/tasks/nginx/proxy.conf

```diff
@@ -1,13 +1,13 @@
 server {
-    server_name idea.gear-tech.io www.idea.gear-tech.io;
+    server_name 13.56.230.75;
     root /var/www/html/;
     index index.html index.htm;
 
-    listen [::]:443 ssl;
-    listen 443 ssl; 
-    ssl_certificate /etc/nginx/cert/cert.crt;
-    ssl_certificate_key /etc/nginx/cert/private.key;
-    ssl_dhparam /etc/nginx/cert/dhparam.pem;        
+    # listen [::]:443 ssl;
+    # listen 443 ssl; 
+    # ssl_certificate /etc/nginx/cert/cert.crt;
+    # ssl_certificate_key /etc/nginx/cert/private.key;
+    # ssl_dhparam /etc/nginx/cert/dhparam.pem;        
```

Comment:
Why SSL has been disabled in Nginx?

Reply:
It has to be deleted. I don't know why this file was here